File: Application/SkuciSe/app/src/main/java/com/example/skucise/models/HelperModels/UserLogin.kt
package com.example.skucise.models.HelperModels

import java.io.Serializable

data class UserLogin(


    val id: Long?,
    val email: String,
    val password: String,
    val role: Byte?,
    val token: String?,
    val fcmToken: String?
): Serializable {}

File: Application/Backend/SkuciSeAPI/SkuciSeAPI/Models/HelperModels/UserLogin.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SkuciSeAPI.Models
{
    public class UserLogin
    {
        public long Id { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public byte Role { get; set; }
        public string Token { get; set; }
        public string FcmToken { get; set; }
    }
}
